exporter/awsemfexporter/datapoint.go

```diff
@@ -164,6 +179,17 @@ func (dps histogramDataPointSlice) CalculateDeltaDatapoints(i int, instrumentati
 	}}, true
 }
 
+func (dps histogramDataPointSlice) IsStaleOrNaN(i int) (bool, pcommon.Map) {
+	metric := dps.HistogramDataPointSlice.At(i)
+	if metric.Flags().NoRecordedValue() {
+		return true, metric.Attributes()
+	}
+	if math.IsNaN(metric.Max()) || math.IsNaN(metric.Sum()) || math.IsNaN(metric.Min()) {
```

Comment:
q: do we also need to check on `count` as well?

Reply:
Could should be an integer and thus we don't need to check it. https://github.com/open-telemetry/opentelemetry-proto/blob/main/opentelemetry/proto/metrics/v1/metrics.proto#L412-L415